src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -360,4 +362,20 @@ private void createDevelopKeystore() throws IOException, InterruptedException {
 
         System.err.println("done creating ks");
     }
+    
+    private String findLatestBuildTool(Path sdkPath) throws IOException {
+        Path buildToolsPath = sdkPath.resolve("build-tools");
+        if (buildToolsPath.toFile().exists()) {
+            Optional<Version> latest = Files.walk(buildToolsPath, 1)
+                    .filter(file -> Files.isDirectory(file) && !file.equals(buildToolsPath))
+                    .map(file -> new Version(file.getFileName().toString()))
+                    .max(Version::compareTo);
+            if (latest.isPresent()) {
```

Comment:
maybe you simplify it to something like `return Files.walk().filter().map().max(...).map(toString).orElseThrow(IOException)`?